adaptivefiltering/utils.py

```diff
@@ -1,4 +1,11 @@
 import collections
+from os import pipe
+from PIL import Image, ImageChops, ImageSequence
```

Comment:
The general style of the project is to do `import PIL` -> use as `PIL.Image`. I know that this my very own personal preference, but sticking to one style consistently is definitely worth doing. Same applies to the next two lines.